### DISCUSSION TITLE
How to use sphinx extensions?

### DISCUSSION THREAD

**Discussion by kkoogqw** [executablebooks/meta#290] [Q&A] [answered]
I tried to test the example in section **Custom Sphinx extensions**  in the guide book. While using the extension  `sphinx-inline-tabs`, I followed the documents with `pip install sphinx-inline-tabs` and add the extension in the file `_config.yml`:

```
sphinx:
  extra_extensions          :
    - sphinx-inline-tabs   
```
but when building the book, it threw an error:

```sh
Extension error:
Could not import extension sphinx-inline-tabs (exception: No module named 'sphinx-inline-tabs')
Traceback (most recent call last):
  File "/home/gqw-wsl/anaconda3/lib/python3.8/site-packages/sphinx/registry.py", line 409, in load_extension
    mod = import_module(extname)
  File "/home/gqw-wsl/anaconda3/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sphinx-inline-tabs'
```
but use the command `pip list` I can see the package:
```
sphinx-inline-tabs                 2021.3.19b5
```
I am not sure if I missed some necessary steps,  and no similar problems were found in the issues. 
So I wander how I can use sphinx extensions correctly. Thanks!

**Comment by chrisjsewell** [comment, marked as answer]
Heya, yes you still have to install the package into your python environment

**Reply by chrisjsewell** [reply]
oh no oops sorry its because it should be:

```yaml
sphinx:
  extra_extensions:
    - sphinx_inline_tabs   

```

**Reply by kkoogqw** [reply]
Oh I'm so stupid. I got it! Thx!!! 😀
